<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tw</class>
 <widget class="QMainWindow" name="tw">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Whitebox Media Library Tools</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="2" rowspan="9">
     <widget class="QWidget" name="tool_window_controls" native="true">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QFrame" name="frame">
      <property name="maximumSize">
       <size>
        <width>125</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Tools</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="show_rename">
         <property name="text">
          <string>Rename Show</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="build_index">
         <property name="text">
          <string>locate files</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="dbg">
         <property name="text">
          <string>File Reconciliation</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="show_images">
         <property name="text">
          <string>Find Images</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="replace">
         <property name="text">
          <string>replace file</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="8" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="1" rowspan="9">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
